name.abuchen.portfolio/src/name/abuchen/portfolio/model/ImageManager.java

```diff
@@ -22,12 +22,37 @@ private ImageManager()
     {
     }
 
+    /**
+     * Retrieves an image associated with the given Attributable and AttributeType.
+     * If not found, a default image is returned.
+     *
+     * @implNote For some time, Windows cut off the logos on the left.
+     *           An offset was added with #2752 In the meantime the problem seems to be fixed,
+     *           so we set the offset to 0 again.
+     *
+     * @see <a href="https://github.com/portfolio-performance/portfolio/pull/2752">#2752</a>
```

Comment:
Let's remove this comment. If one wants to have a look at the history, then one can use git. We should have comments only for "strange" workarounds which are in the code, not for the ones we do not need (anymore).


The commit message can include the explanation that it is not needed anymore.

Reply:
Done